docs/en/concepts-and-designs/kubesphere-logging.md

```diff
@@ -0,0 +1,46 @@
+# KubeSphere Logging
+
+## Overview
+
+The KubeSphere logging system comprises many components that work together to achieve centralized log management. Once you enable the logging module during installation, a few components will be deployed or created within the namespace `kubesphere-logging-system`:
+
+- Fluent Bit: log collector deployed on each node.
+- Fluent-bit operator: automates tasks for operating Fluent bit instances.
+- Logsidecar injector: injects sidecar into pods for streaming on-disk files to stdout.
+- Elasticsearch cluster: log storage backend.
+- Curator: performs stale log cleanup.
+
+The diagram below presents the overall logging architecture:
+
+![](../../images/kubesphere-logging-overview.png)
+
+The following sections will detail the design and functionality of each component. 
+
+## Log Collection and Forward
+
+There are two types of logs produced by containers:
+
+- stdout and stderr: written to `.log` files in the host directory `/var/log/containers` by default.
+- application logs: files on pod volume, file locations may vary. 
+
+Fluent Bit serves as the node logging agent. It has access to container log directory and works for applications' stdout and stderr. In KubeSphere, we forked and modified the origin Fluent Bit project fluent/fluent-bit. The difference between fluent/fluent-bit and [kubesphere/fluent-bit](https://github.com/kubesphere/fluent-bit) lies that we add a reload interface to support dynamic configuration (see issue [#365](https://github.com/fluent/fluent-bit/issues/365)).
+ 
+As shown in the diagram below, Fluent-bit controller implements the reload interface and is responsible for the life cycle of fluent-bit process.
+
+![](../../images/kubesphere-logging-fluentbit.png) 
+
+The approach to collect on-disk logs is using a streaming sidecar. The sidecar container streams application logs to its own stdout. In this way, we manage log collection in concert. Besides, [Logsidecar Injector](https://github.com/kubesphere/logsidecar-injector) comes to automate sidecar injection. It takes advantages of `MutatingAdmissionWebhook` to control pod creation and modify pod templates.
```

Comment:
"a streaming sidecar" => "a log forwarder sidecar container"?
"The approach to collect on-disk logs " => "The approach to collect container logs output to a file on a pod's volume"
"The sidecar container streams application logs to its own stdout" => "The sidecar container forwards application logs on disk to its own stdout"
"we manage log collection in concert" => "KubeSphere manages log collection in an unified way"
"It takes advantages of `MutatingAdmissionWebhook` to control pod creation and modify pod templates." => "It takes advantages of `MutatingAdmissionWebhook` to control the log forwarder sidecar injection during pod creation ."